### PR TITLE
Set up React Application

### PR DESCRIPTION
**Pull request by erikwesterberg**
### User Story
```
As a development team,
In order to be able to start adding features,
We would like to have a basic application structure.
```

### Feature description
https://www.pivotaltracker.com/story/show/168248826

### Description of PR
- React scaffolding Application
- Clean up codebase

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```
Comment (pedbr) [inline .gitignore]: Delete comments here.

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
```
Comment (pedbr) [inline public/index.html]: Delete comments here.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (pedbr) [inline public/index.html]: Delete comments

File: src/serviceWorker.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    });
+}
+
+export function unregister() {
+  if ('serviceWorker' in navigator) {
+    navigator.serviceWorker.ready.then(registration => {
+      registration.unregister();
+    });
+  }
+}
```
Comment (pedbr) [inline src/serviceWorker.js]: Remove comments from this file also.

**Review by pedbr — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (erikwesterberg) [reply .gitignore]: fixed

File: public/index.html (hunk 2 above)
Comment (erikwesterberg) [reply public/index.html]: fixed

File: public/index.html (hunk 3 above)
Comment (erikwesterberg) [reply public/index.html]: u commented the same file twice

Comment (erikwesterberg) [reply public/index.html]: fixed

**Review by pedbr — APPROVED** [review]

**Pull request merged**